tamara32/https-github.com-coinbase-toshi
# Fix sync internal (#1)

* Change PUT /transactions to POST /transactions

* Fix docs for relay transaction. Closes #110

* This also missed the UnconfirmedRawTransaction switchover

* Look for POSTed tx in the body

* Allow cross-origin API requests

* Fix additional fields for orphan txs

* Don't let conflicts prevent mempool acceptance of disconnected txs

* Fix unconfirmed balance bug due to orphans/conflicts

* Minor cleanup; add tests

* Fixed GET/POST typo in transactions

* Rewrite Dockerfile on official ruby image

* Ensure advisory lock/unlock are issued on same connection

* Update Dockerfile

* update bitcoin-ruby

* update bitcoin-ruby

  fixes regtest

* Fix slow query for unspent outputs

* API endopoint for balance_at

* Check UNIX timestamp format

* Add 400 Error type

* Add time to the json response

* BadRequest Error

* document bootstrap.dat usage

* Do not insert randomly generated SSH key pair

* Make vagrant SSH configurable

* Remove vagrant SSH configuration options

do not require ssh key

* Update documentation for testing

Include commands to run TheBlueMatt's test suite

* updated docker file to not reference quay.io

* add block height as a parameter to balance_at endpoint

* use docker compose

* update readme

* Replace Vagrant with Fig

* test inside a docker container

* need this to figure out the tests

* rm vagrant, patch up compose

* update readme ; fix tests

* Update eventmachine for Ruby 2.2

* Update faye-websocket

* Add 2.2.0 to Travis coverage

* Implement BIP66

* Fix testnet syncing
